### PR TITLE
[14.0][ADD] account_bank_statement_reopen_skip_undo_reconciliation

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
After installing this module when you reopen a posted bank statement line the
application will not unreconcile any previous reconciled bank statement lines.

This feature improves the usability of the bank statements, as it may be
frequent for a user to adjust the statement lines during the reconciliation
process.

cc @ForgeFlow @alexis-via

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@sebastienbeau after checking https://github.com/akretion/odoo-usability/blob/14.0/account_usability/models/account_bank_statement.py#L46 it does not seem to me that it is really unreconciling, because when setting the move to draft the unreconcile also happens https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_move.py#L2825

At least my tests failed, until I fixed it here.

File: account_bank_statement_reopen_skip_undo_reconciliation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Copyright 2022 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Bank Statement Reopen Skip Undo Reconciliation",
+    "summary": "When reopening a bank statement it will respect the "
+    "reconciled entries.",
+    "version": "14.0.1.0.0",
+    "author": "ForgeFlow, Akretion, Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_reopen_skip_undo_reconciliation/__manifest__.py:12]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```
also above copyrights...

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM!

Can you confirm the copyrights are correct?

**Review by mariadforgeflow — APPROVED** [review]
Code review. LGTM!

**Pull request merged**